cas/aciinfo.go

```diff
@@ -0,0 +1,47 @@
+package cas
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/appc/spec/schema"
+)
+
+// ACIInfo is used to store informations about an ACI
+// Im is the ACI's image manifest. This avoids extracting it from the ACI every
+// time it's needed
+// BlobKey is the key in the blob store of the related ACI file.
+// Time is the time this ACI was imported in the store
+// Latest defines if the ACI was imported using the latest pattern (no version
+// label provided on ACI discovery)
+type ACIInfo struct {
+	Im      *schema.ImageManifest
```

Comment:
The description is great, but I was suggestion an expansion of the struct member name. Maybe something like "ImageManifest" or just "Manifest".